src/dev/i18n/extract_default_translations.js

```diff
@@ -38,19 +38,19 @@ function addMessageToMap(targetMap, key, value) {
 }
 
 export async function extractDefaultTranslations(inputPath) {
-  const entries = await globAsync('*.{js,jsx,jade,ts,tsx,html,hbs,handlebars}', {
+  const entries = await globAsync('*.{js,jsx,pug,jade,ts,tsx,html,hbs,handlebars}', {
     cwd: inputPath,
     matchBase: true,
   });
 
-  const { htmlEntries, codeEntries, jadeEntries, hbsEntries } = entries.reduce(
+  const { htmlEntries, codeEntries, pugEntries, hbsEntries } = entries.reduce(
     (paths, entry) => {
       const resolvedPath = resolve(inputPath, entry);
 
       if (resolvedPath.endsWith('.html')) {
         paths.htmlEntries.push(resolvedPath);
-      } else if (resolvedPath.endsWith('.jade')) {
-        paths.jadeEntries.push(resolvedPath);
+      } else if (resolvedPath.endsWith('.jade') || resolvedPath.endsWith('.pug')) {
```

Comment:
since switching to `pug`, I would remove `jade` at all for avoiding misunderstanding